apps/bundle-size/src/commands/build.js

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+const chalk = require('chalk');
+const Table = require('cli-table3');
+const fs = require('fs-extra');
+const glob = require('glob');
+const path = require('path');
+const prettyBytes = require('pretty-bytes');
+
+const buildFixture = require('../utils/buildFixture');
+const { hrToSeconds } = require('../utils/helpers');
+const prepareFixture = require('../utils/prepareFixture');
+
+/**
+ * @param {{ verbose: boolean }} options
+ */
+async function build(options) {
```

Comment:
this command isn't actually building the package right ? just the fixtures 

I think a name like `measure` would be more appropriate ?

Reply:
It _builds_ fixtures, but `measure` is also fine for me.

Reply:
if we decide to change the name, pls don't forget to rename files and related stuff

Reply:
Renamed to `measure`, all files and scripts were renamed in `51b8acb`